doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,11 +8,18 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
+:class:`~ray.train.ScalingConfig` overview
+------------------------------------------
+Ray Train's :class:`~ray.train.ScalingConfig` configures the number of training workers
+and the resources they should use.
 
-The scaling configuration specifies distributed training properties like the number of workers or the
-resources per worker.
+A *worker* here refers to a *training worker*. Technically, it's a
+:ref:`Ray Actor <actor-key-concept>` that runs your training loop.
+
+The :class:`Trainer <ray.train.trainer.BaseTrainer>` object you instantiate in the
+training script contains the settings to run your training. When you call
+:meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`, it will be scheduled
+as a :ref:`Ray Actor <actor-key-concept>`. It can then also use resources.
 
 The properties of the scaling configuration are :ref:`tunable <tune-search-space-tutorial>`.
```

Comment:
Looking this over again, can we remove this and just mention it in the hyperparameter tuning guide? IMO we definitely do no need to put this in the introduction haha.

Reply:
I've moved it in a note, lmkwyt

Reply:
Actually, removed after moving the example to the docstring